Documentation/data-collection.md

```diff
@@ -21,318 +37,495 @@ data:
     #
+    # (Insights operator team)
+    #
     # insightsclient_request_send tracks the number of metrics sends.
     - '{__name__="insightsclient_request_send_total"}'
     #
+    # (@openshift/openshift-team-app-migration)
+    #
     # cam_app_workload_migrations tracks number of app workload migrations
     # by current state. Tracked migration states are idle, running, completed, and failed.
     - '{__name__="cam_app_workload_migrations"}'
     #
+    # (uznknown)
+    #
     # cluster:apiserver_current_inflight_requests:sum:max_over_time:2m gives maximum number of requests in flight
     # over a 2-minute window. This metric is a constant 4 time series that monitors concurrency of kube-apiserver and
     # openshift-apiserver with request type which can be either 'mutating' or 'readonly'.
     # We want to have an idea of how loaded our api server(s) are globally.
     - '{__name__="cluster:apiserver_current_inflight_requests:sum:max_over_time:2m"}'
     #
-    # (monitoring-team) cluster:telemetry_selected_series:count tracks the total number of series
+    # (@openshift/openshift-team-monitoring)
+    #
+    # cluster:telemetry_selected_series:count tracks the total number of series
     # sent via telemetry from each cluster.
     - '{__name__="cluster:telemetry_selected_series:count"}'
     #
-    # (monitoring-team) openshift:prometheus_tsdb_head_series:sum tracks the total number of active series
+    # (@openshift/openshift-team-monitoring)
+    #
+    # openshift:prometheus_tsdb_head_series:sum tracks the total number of active series
     - '{__name__="openshift:prometheus_tsdb_head_series:sum"}'
     #
-    # (monitoring-team) openshift:prometheus_tsdb_head_samples_appended_total:sum tracks the rate of samples ingested
+    # (@openshift/openshift-team-monitoring)
+    #
+    # openshift:prometheus_tsdb_head_samples_appended_total:sum tracks the rate of samples ingested
     # by prometheusi.
     - '{__name__="openshift:prometheus_tsdb_head_samples_appended_total:sum"}'
     #
-    # (monitoring-team) monitoring:container_memory_working_set_bytes:sum tracks the memory usage of the monitoring
+    # (@openshift/openshift-team-monitoring)
+    #
+    # monitoring:container_memory_working_set_bytes:sum tracks the memory usage of the monitoring
     # stack.
     - '{__name__="monitoring:container_memory_working_set_bytes:sum"}'
     #
-    # (monitoring-team) monitoring:haproxy_server_http_responses_total:sum tracks the number of times users access
+    # (@openshift/openshift-team-monitoring)
+    #
+    # monitoring:haproxy_server_http_responses_total:sum tracks the number of times users access
     # monitoring routes.
     - '{__name__="monitoring:haproxy_server_http_responses_total:sum"}'
     #
-    # (rhmi, @openshift/openshift-team-cluster-manager) rhmi_status reports the status of an RHMI installation.
+    # (https://github.com/integr8ly, @david-martin)
```

Comment:
we should also reach out to the team responsible for `integr8ly` to have either a team created or a team name

Reply:
the problem with teams outside of the openshift org is that they're not public so you can't `@` anybody. IMHO the clearer option is to link to the organization directly.